### ISSUE TITLE
Bug: dfgprojpapers dumps whole bibliography

### ISSUE THREAD

**Issue by miili** [KWARC/LaTeX-proposal#36]
Hi there,

when using `\dfgprojpapers[articles]{cat_2017,parrot_2018}` the all referenced citations are dumped regardless of the `{selectors}`.

From the documentation I suppose that this is not the intended behaviour?

Thanks for this really helpful template! :)

Marius

Related? #30

**Comment by kohlhase** [comment]
> From the documentation I suppose that this is not the intended behaviour?

indeed that is not the intended behavior; an interesting effect. I am investigating (but baffled)

**Comment by miili** [comment]
compiled on two different systems now, seeing the same behavior on both.

**Comment by kohlhase** [comment]
yes, but the `[articles]` is intended behavior, which is supposed to only select the articles. from the list of project papers (useful if you want to have a bibliography classified by type. 
So the problem still persists.
